tests/test_common_gpu.py

```diff
@@ -1135,6 +1135,20 @@ def test_dora_ephemeral_gpu_offload(self):
         # The results should be the same
         assert torch.allclose(out_peft_model_cpu, out_peft_model_ego)
 
+    @require_torch_gpu
```

Comment:
Could you please move this whole test to `tests/test_gpu_examples.py`? Please create a new test class `TestBOFT` at the very bottom of the file and place this test inside of the class. You could also split it into two tests, one for `Linear` and one for `Conv2d`.